### PR TITLE
[13.0][IMP] maintenance_plan: Set planned_hours in request from plan if maintenance_timesheet is installed.

### PR DESCRIPTION
**Pull request by victoralmau**
Set `planned_hours` in request from plan if `maintenance_timesheet` is installed.

Please @pedrobaeza and @CarlosRoca13 can you review it?

@Tecnativa

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Aren't you mixing `maintenance_plan` and `maintenance_timesheet`?

**Comment by victoralmau** [comment]
> Aren't you mixing `maintenance_plan` and `maintenance_timesheet`?

To prevent a a glue module, I have set the method to be used only if `maintenance_plan` is installed. Do you prefer to do it another way?

**Comment by pedrobaeza** [comment]
That doesn't serve, Víctor. You can't call super over a method that is not defined on its ancestors in the dependency chain. It should be the contrary: in `maintenance_plan`, if `planned_hours` exist, it's filled.

**Comment by victoralmau** [comment]
> That doesn't serve, Víctor. You can't call super over a method that is not defined on its ancestors in the dependency chain. It should be the contrary: in `maintenance_plan`, if `planned_hours` exist, it's filled.

Ok, changes done.

File: maintenance_plan/models/maintenance_equipment.py [hunk 1]
```diff
@@ -80,6 +81,9 @@ def _prepare_request_from_plan(self, maintenance_plan, next_maintenance_date):
             "note": maintenance_plan.note,
             "company_id": maintenance_plan.company_id.id or self.company_id.id,
         }
+        if "planned_hours" in request_model._fields:
```
Comment (pedrobaeza) [inline maintenance_plan/models/maintenance_equipment.py:85]: You always forget to comment why doing these things that will serve in the future. In this case, a comment saying that this field comes from `maintenance_timesheet` and for avoiding a glue module just for that.

**Comment by victoralmau** [comment]
Ping @pedrobaeza

**Review by pedrobaeza — APPROVED** [review]

**Review by CarlosRoca13 — APPROVED** [review]
LGTM

**Pull request merged**